test/commands/commit.spec.js

```diff
@@ -157,9 +161,26 @@ describe('commit command', () => {
       })
     })
 
+    describe('when rebasing', () => {
+      it('should cancel the hook', async () => {
+        // Use an exception to suspend code execution to simulate process.exit
+        mockProcess.mockProcessExit(new Error('ProcessExit0'))
```

Comment:
It would be cool to "simulate" the "rebase" to test the `cancelIfRebasing` code 😊

Reply:
that's more or less what I'm doing when I mock ` fs.existsSync` to simulate detection of rebase